Duplicati/Library/Utility/ReadLimitLengthStream.cs

```diff
@@ -50,8 +92,24 @@ public override int Read(byte[] buffer, int offset, int count)
         }
 
         public override long Seek(long offset, SeekOrigin origin)
-        {
-            return m_innerStream.Seek(offset, origin);
+        {
+            long innerPosition;

```

Comment:
If `innerPosition` is no longer used, can we remove it?

Reply:
Done.